R/composition.R

```diff
@@ -170,10 +168,9 @@ lift_vd <- function(..f, ..., .type) {
 #'
 #' @export
 #' @examples
-#' \dontrun{ %spacing
+#' #
```

Comment:
do you know of a better way to ensure that Roxygen leaves some whitespace when it collates the examples together?